org.kde.kdiff3.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "org.kde.kdiff3",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kdiff3",
+    "finish-args": [
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--socket=system-bus",
```

Comment:
please establish wich dbus name it needs to access instead of open the system bus wide.

Reply:
This is weird. I don't think this needs bus access.